README.md

```diff
@@ -35,9 +35,20 @@ To ensure the service is running, you must declare it in your application config
    {"name": "fdc3"}
 ]
 ```
+During development, you can add a URL for specifying a custom location or a specific version:
 
-### Import the Client API
+```
```

Comment:
If you add `json` it will syntax highlight it.

Reply:
* mind-blown-gif *

✅ Done for all of the projects